app/controllers/movies_controller.rb

```diff
@@ -1,5 +1,24 @@
 class MoviesController < ApplicationController
   def index
-    @movies = Movie.all
+    @movies = []
+    if params[:search] == 'top_movies'
+      conn = Faraday.new('https://api.themoviedb.org')
+      response1 = conn.get("/3/movie/popular?api_key=#{Figaro.env.movie_api_key}&page=1")
+      response2 = conn.get("/3/movie/popular?api_key=#{Figaro.env.movie_api_key}&page=2")
```

Comment:
let's change Figaro.env.movie_api_key to ENV['movie_api_key'] per today's lesson